app/javascript/components/student/SolutionSummary.tsx

```diff
@@ -33,21 +33,21 @@ export type SolutionSummaryRequest = {
   }
 }
 
-export type SolutionSummarySolution = {
-  id: string
-  hasMentorDiscussionInProgress: boolean
-  hasMentorRequestPending: boolean
-  completedAt?: string
+export type Track = {
+  title: string
+  medianWaitTime: string
```

Comment:
What is the unit in which the wait time is measured? It might be nice to add the unit to the name to reduce confusion.

Reply:
That's a good question. I actually think we should format this server-side, so this is actually just a string that should be rendered by JS rather than anything intelligent. So it's probably named ok?

Reply:
Then it's perfectly fine.